simtools/utils/names.py

```diff
@@ -557,210 +534,62 @@ def layout_telescope_list_file_name(name, label):
     return file_name
 
 
-def ray_tracing_file_name(
+def generate_file_name(
+    file_type,
+    suffix,
     site,
     telescope_model_name,
-    source_distance,
     zenith_angle,
-    off_axis_angle,
-    mirror_number,
-    label,
-    base,
+    azimuth_angle=None,
+    off_axis_angle=None,
+    source_distance=None,
+    mirror_number=None,
+    label=None,
+    extra_label=None,
 ):
     """
-    File name for files required at the RayTracing class.
+    Generate a file name for output, config, or plotting.
+    Used e.g., to generate camera-efficiency and ray-tracing output files.
 
     Parameters
     ----------
+    file_type: str
+        Type of file (e.g., config, output, plot)
+    key: str
```

Comment:
This should be named suffix. Description is not matching.

Reply:
Exactly. Changed it.